### PR TITLE
Add jenkins build step script

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by nghiant2710** [comment]
@petrosagg , please check this PR

File: automation/jenkins_build.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Set env var
+ACCESS_KEY=$1
```
Comment (petrosagg) [inline automation/jenkins_build.sh]: Can read those vars from the environment directly instead of passing them as parameters?

Comment (nghiant2710) [inline automation/jenkins_build.sh]: Sure, by exporting them as environment variables. Will still have to export them in jenkins configuration shell script

**Comment by nghiant2710** [comment]
@petrosagg ^

**Pull request merged**